FWCore/Framework/src/EventSetupsController.cc

```diff
@@ -40,15 +46,22 @@ namespace edm {
       // EventSetupsController and in the EventSetupProvider
       fillEventSetupProvider(*this, *returnValue, iPSet);
 
+      numberOfConcurrentIOVs_.initialize(optionsPset);
```

Comment:
Let's put all the parameters related to the IOVs into one PSet and only pass that PSet to this class.

Reply:
Done.